torch/csrc/distributed/c10d/init.cpp

```diff
@@ -2556,6 +2541,27 @@ options :class:`~torch.distributed.ProcessGroupNCCL.Options`).
               &::c10d::Backend::endCoalescing,
               py::call_guard<py::gil_scoped_release>());
 
+  // base Backend::Options binding
```

Comment:
Maybe add a TODO for thhe consolidation of this with `DistributedBackendOptions` that we discussed before?